src/modules/75-cf/components/FlagActivation/__tests__/FlagPrerequisites.test.tsx

```diff
@@ -55,4 +55,61 @@ describe('FlagPrerequisites', () => {
 
     expect(screen.getByTestId('flag-prerequisite-modal')).toMatchSnapshot()
   })
+
+  test('form should render when adding prerequisite', async () => {
+    renderComponent()
+    await waitFor(() => {
+      expect(screen.getByText('cf.featureFlags.newPrerequisite')).toBeInTheDocument()
+    })
+
+    userEvent.click(screen.getByText('cf.featureFlags.newPrerequisite'))
+
+    await waitFor(() => expect(screen.getByTestId('flag-prerequisite-modal')).toBeInTheDocument())
+
+    userEvent.click(screen.getByTestId('prerequisites'))
+
+    await waitFor(() => expect(screen.getByTestId('prerequisites-form')).toBeInTheDocument())
+  })
+
+  test('it should render feature flags in dropdown', async () => {
+    renderComponent()
+    await waitFor(() => {
+      expect(screen.getByText('cf.featureFlags.newPrerequisite')).toBeInTheDocument()
+    })
+
+    userEvent.click(screen.getByText('cf.featureFlags.newPrerequisite'))
+    await waitFor(() => expect(screen.getByTestId('flag-prerequisite-modal')).toBeInTheDocument())
+    userEvent.click(screen.getByTestId('prerequisites'))
+
+    await waitFor(() => expect(screen.getByTestId('prerequisites-form')).toBeInTheDocument())
+
+    expect(screen.getByTestId('prerequisites-variations-dropdown-0')).toBeInTheDocument()
+    userEvent.click(screen.getByTestId('prerequisites-dropdown-0'))
+    mockFeature.prerequisites?.forEach(prerequisite => {
+      expect(screen.getByText(prerequisite.feature))
```

Comment:
should this be asserting something? e.g:

```suggestion
      expect(screen.getByText(prerequisite.feature)).toBeInTheDocument()
```

?

Reply:
Added this in :)